projects/observability/src/shared/components/span-detail/span-data.ts

```diff
@@ -11,6 +11,7 @@ export interface SpanData {
   requestBody: string;
   responseHeaders: Dictionary<unknown>;
   responseCookies: Dictionary<unknown>;
+  responseCookiesMetadata: Dictionary<Dictionary<unknown>>;
```

Comment:
should this be optional?

Reply:
since other related fields aren't optional, making this also of the same category.